pkg/services/policer/process.go

```diff
@@ -72,6 +79,13 @@ func (p *Policer) shardPolicyWorker(ctx context.Context) {
 				}
 			}
 		}
+
+		select {
+		case <-ctx.Done():
+			return
+		case <-t.C:
```

Comment:
But it's not per-object, isn't it?

Reply:
yes, it is per object replication (per 10 objects for now but can be per any number of objects (e.g. 1)). do you want to fetch objects at once and then replicate them with sleeps? e.g. read 100 objects and replicate the last one after 99+ seconds? what if the batch size is 1000?

Reply:
OK, let's try this way.